src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -145,14 +145,12 @@ private ReadFilterLibrary(){ /*no instance*/ }
         @Override public boolean test(final GATKRead read) {
             return read.isProperlyPaired();}}
 
-    //TODO: should this reject supplementary alignments ? If not it should be
-    //removed since its redundant with NotSecondaryAlignmentReadFilter
-    //https://github.com/broadinstitute/gatk/issues/2165
+    //* this now tests for unmapped or neither secondary nor supplementary as its name suggests */
```

Comment:
So I'm not sure this PR is the right place to address this issue/ I'm not sure we're in agreement about changing PrimaryAlignmentFilter.  It's confusing because the sam spec refers to `primary` as meaning `not secondary`, which doesn't quite jibe with common usage as `not secondary AND not supplementary`.  I think it's best if we at the very least address this in a separate PR (which needs a matching PR in protected to fix tools that intended to use the original meaning.)